client/src/components/Header.js

```diff
@@ -7,10 +7,21 @@ import Select from '@material-ui/core/Select';
 import MenuItem from '@material-ui/core/MenuItem';
 import Button from '@material-ui/core/Button';
 import useStyles from './HeaderStyles';
+import Login from './login/Login';
 
 export default function Header() {
   const classes = useStyles();
 
+  const [showModal, setShowModal] = React.useState(false);
+  const handleOpen = () => {
+    setShowModal(true);
+    document.body.classList.add(classes.disable);
```

Comment:
no, you shouldn't work on a DOM directly (document.body) in most of the cases
use classnames looking like: className=classnames('class-name', { disabled: showModal })..